rpcs3/emucore.vcxproj

```diff
@@ -516,7 +516,7 @@
     <ClInclude Include="Emu\Cell\lv2\sys_uart.h" />
     <ClInclude Include="Emu\Cell\lv2\sys_bdemu.h" />
     <ClInclude Include="Emu\Cell\lv2\sys_console.h" />
-    <ClCompile Include="Emu\Cell\lv2\sys_btsetting.h" />
+    <ClCompile Include="Emu\Cell\lv2\_sys_btsetting.h" />

```

Comment:
this should be a ClInclude instead of ClCompile so it doesnt attempt to compile a .h file

Reply:
Thank you very much, the reason for all this mess is that I wanted to group all lv2/sys_x files added with the others because VS didn't do that.
And I made a typo.